portaljs-pages/page-admin-user-details/resources/page.properties

```diff
@@ -1,5 +1,5 @@
 displayName=Bonita Admin User Details
 name=custompage_adminUserDetailsBonita
-description=Default admin user details
+description=This page provides detail information about a user. It is dedicated to Admin-like profiles. This page has not been created with the UI Designer.
```

Comment:
```suggestion
description=This page provides detailed information about a user. It is dedicated to Admin-like profiles. This page has not been created with the UI Designer but it can still be used as a custom page in any application.
```